packager/import.go

```diff
@@ -12,38 +11,31 @@ import (
 	"github.com/docker/docker/pkg/archive"
 )
 
-var (
-	// ErrNoArtifactsDirectory indicates a missing artifacts/ directory
-	ErrNoArtifactsDirectory = errors.New("No artifacts/ directory found")
-)
-
 // Importer is responsible for importing a file
 type Importer struct {
 	Source      string
 	Destination string
 	Loader      loader.BundleLoader
-	Verbose     bool
 }
 
 // NewImporter creates a new secure *Importer
 //
 // source is the filesystem path to the archive.
 // destination is the directory to unpack the contents.
 // load is a loader.BundleLoader preconfigured for loading bundles.
-func NewImporter(source, destination string, load loader.BundleLoader, verbose bool) (*Importer, error) {
+func NewImporter(source, destination string, load loader.BundleLoader, verbose bool) *Importer {
```

Comment:
We don't want to verbose anymore? Should we keep the verbose parameter then?

Reply:
Great catch; thank you!  I'll remove.